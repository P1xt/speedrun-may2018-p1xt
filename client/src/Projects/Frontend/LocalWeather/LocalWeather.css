.LocalWeather {
  text-align: left;
  margin-top: 40px; }
  .LocalWeather > h1 {
    text-transform: none; }
