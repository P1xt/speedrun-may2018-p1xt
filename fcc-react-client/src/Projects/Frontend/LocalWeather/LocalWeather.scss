.LocalWeather {
  text-align: center;
  margin-top: 40px;

  &>h1 {
    text-transform: none
  }
  >.card {
    max-width: 400px;
    width: 400px;
    margin: 10px auto;
    display: block;
    .card-subtitle {
      margin-top: 20px;
    }
    .card-text {
      min-height: 70px;
      margin-top: 20px;
    }
  }
  &_buttons {
    display: flex;
    flex-direction: row;
    justify-content: space-around;
  }
}
