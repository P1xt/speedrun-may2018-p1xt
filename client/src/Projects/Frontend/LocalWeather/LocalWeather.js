import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./LocalWeather.css";

const LocalWeather = () => {
  return (
    <div className="LocalWeather">
      <Container>
        <Row>
          <Col>
            <h1>Local Weather</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default LocalWeather;
