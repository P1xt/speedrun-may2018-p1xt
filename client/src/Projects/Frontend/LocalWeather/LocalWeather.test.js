import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import LocalWeather from "./LocalWeather";

describe("<LocalWeather />", () => {
  it("renders without crashing", () => {
    shallow(<LocalWeather />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<LocalWeather />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
