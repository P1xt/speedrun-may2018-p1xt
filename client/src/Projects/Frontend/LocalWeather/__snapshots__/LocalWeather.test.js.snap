// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocalWeather /> renders correctly 1`] = `
<div
  className="LocalWeather"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Local Weather
        </h1>
      </div>
    </div>
  </div>
</div>
`;
